SYCL/ESIMD/api/functional/value_conv.hpp

```diff
@@ -51,15 +52,22 @@ std::vector<SrcT> generate_ref_conv_data() {
 
   // TODO: Implement functions for obtain lowest and max values without UB
   // cases.
-  static const SrcT nan = value<SrcT>::nan();
-  static const SrcT inf = value<SrcT>::inf();
-  static const SrcT denorm = value_conv<SrcT, DstT>::denorm_min();
   static const SrcT positive = static_cast<SrcT>(126.75);
+  static const SrcT max = 10;
+  // Use zero for unsigned types
+  static const SrcT min =
+      std::min(static_cast<SrcT>(-max), static_cast<SrcT>(0));
```

Comment:
```suggestion
  static const SrcT max = 10;
  // Use zero for unsigned types
  static const SrcT min =
      std::min(static_cast<SrcT>(-max), 0);
```

Reply:
when I try to apply this comment I got the following compilation error:
```
error: no matching function for call to 'min'
      std::min(static_cast<SrcT>(-max), 0);
note: candidate template ignored: deduced conflicting types for parameter '_Ty' ('float' vs. 'int')
_NODISCARD _Post_equal_to_(_Right < _Left ? _Right : _Left) constexpr const _Ty&(min)(
                                                                                 ^
```

Reply:
```suggestion
  static const SrcT max = 10;
  // Use zero for unsigned types
  static const SrcT min = std::min<SrcT>(-max, 0);
```

Reply:
this comment was applied in this commit: 33d800706e8c6edfd581023acf7402823d73d8c6